app/controllers/katello/remote_execution_controller.rb

```diff
@@ -0,0 +1,48 @@
+module Katello
+  class RemoteExecutionController < JobInvocationsController
```

Comment:
Looking further down the diff, it appears this is providing a controller to hit remotely but not any UI components itself?